core-field-editors/Makefile

```diff
@@ -0,0 +1,7 @@
+export PATH := ./node_modules/.bin:${PATH}
+
+compile:
```

Comment:
Hi, Couldn't this also be moved to the package.json as an npm script, plus npm will detect the right binary to use without specifying explicitly the PATH